handbook/engineering/hiring/software-engineer-coding-exercise.md

```diff
@@ -53,6 +53,16 @@ This exercise is designed to give you an opportunity to demonstrate your skills
 - You can look up documentation on the internet while you are coding.
 - After three hours, you will email us your solution as a zip file and we will get back to you on next steps within 2 business days.
 
+## Security coding exercise
+
+We designed this exercise to give you an opportunity to demonstrate your software development and collaboration skills, as they come to developing a small, secure application.
+
+- You will choose a 90-minute timeframe to work with a member of our team on a small console application.
```

Comment:
This makes it sound like the exercise is synchronous with a Sourcegraph interview, but reading the other points it seems like it isn't. What is your intent?

Reply:
It wasn't. I've pushed an update that is more clear.